Sources/Instrumentation/URLSession/InstrumentationUtils.swift

```diff
@@ -48,4 +51,21 @@ enum InstrumentationUtils {
             ptr = ptr.successor()
         }
     }
+
+    static func usesUndocumentedAsyncAwaitMethods() -> Bool {
+#if os(macOS)
+        let os = ProcessInfo.processInfo.operatingSystemVersion
+        if os.majorVersion >= 13 {
+            return true
+        }
+#else
+        let version = UIDevice.current.systemVersion
+        if let versionNumber = Double(version),
```

Comment:
Would https://forums.swift.org/t/swift-concurrency-back-deployment/51908/13 be preferred to check for concurrency availability?

Reply:
It only fails in iOS 16, previous versions work with async/await perfectly

Reply:
wow, bizarre! good spot